source/river/assets/test-config.kdl

```diff
@@ -31,6 +31,41 @@ services {
             "0.0.0.0:4443" cert-path="./assets/test.crt" key-path="./assets/test.key" offer-h2=true
         }
 
+        // Apply Rate limiting to this service
```

Comment:
Hey @branlwyd, I'd be interested in your take on this configuration format (and whether these docs make sense)!